Polycube CI with GitHub Actions (#364)
polycube-network/polycube: Documentation/developers/codegen.rst

Among the services that are automatically provided, the C++ skeleton handles the case in which a complex object is requested, such as the entire configuration of the service, which is returned by disaggregating the big request into a set of smaller requests for each *leaf* object.
Hence, this leaves to the programmer only the responsibility to implement the interaction with *leaf* objects.

The easiest way to generate a stub is to use the ``polycubenets/polycube-codegen`` docker image.
This contains ``pyang``, ``swagger-codegen`` and a script to invoke them.

> In order to run the image, it is needed to mount a volume into ``/polycube-base-datamodels`` where the base datamodels are located on the host (by default ``/usr/local/include/polycube/datamodel-common/``), it is also needed to mount a volume to share the input yang model and the output folder.

Reviewer: ```suggestion
In order to run the image, you need to mount a volume into ``/polycube-base-datamodels`` where the base datamodels are located on the host (by default ``/usr/local/include/polycube/datamodel-common/``).
Furthermore, you need also to mount a volume to share the input yang model and the output folder.
```
